src/awesome_map/BoundaryInterceptor.js

```diff
@@ -232,10 +322,18 @@ define(function(require) {
                 this._touchState = targetState;
                 break;
 
-            case EventTypes.DRAG:
             case EventTypes.DRAG_END:
+                /* jshint -W086 */// Expected break statement 
+
+                this._debouncedDetermineBoundaryVisibility();
```

Comment:
Mostly just to be consistent, but you're right it probably doesn't have to wait, it can just update right away.